src/ol_concourse/pipelines/open_edx/mfe/pipeline.py

```diff
@@ -159,7 +159,7 @@ def mfe_job(
         and OpenEdxMicroFrontend[mfe_name].value == OpenEdxMicroFrontend.learn.value
     ):
         mfe_smoot_design_overrides = """
-        npm pack @mitodl/smoot-design@^3.4.0
+        npm pack @mitodl/smoot-design@^5.0.0
```

Comment:
In `src/bridge/settings/openedx/mfe/slot_config/mitxonline/learning-mfe-config.env.jsx`, could you update the import to
```ts
import * as remoteTutorDrawer from "./mitodl-smoot-design/dist/bundles/remoteTutorDrawer.umd.js"

remoteTutorDrawer.init({ /* etc */})
```

In the bump from 3 to 5 we renamed the export since it's no longer just AiChat

Reply:
Thanks for pointing that out.